### ISSUE TITLE
Feature Request: quota warning more prominent

### ISSUE THREAD

**Issue by jbrrrr** [nextcloud/desktop#892]
One of our users asked for a more prominent quota warning:

As far I know: At the moment a user which uses only a sync client (no browser access to cloud) realizes only at the moment when the quota is reached by looking into sync client activities that quota is exceeded.

Another more prominent (more eye catching, less ignorable and earlier f.ex. 95% reached) way to inform the non expert sync client only user about his actual quota usage would be great.

**Comment by lachmanfrantisek** [comment]
You can try [Quota warning app](https://apps.nextcloud.com/apps/quota_warning). You can get a notification and optionally email about the reached limits...

**Comment by jbrrrr** [comment]
Thank you for this hint! I will try that - Seems to be a simple(r) solution.

**Comment by camilasan** [comment]
What do you think @jancborchardt?

**Comment by jancborchardt** [comment]
Yes, this is a good point. We should probably display a notification at 80 % of storage use and/or at 95 % of use.

**Comment by barisbasar1209** [comment]
Hi, 

@joshtrichards since you updated things here recently I hope it is fine that I direct this post to you. 

Has this problem been fixed? 
I just tried to replicate this, however without any success. I run the Windows client version 3.15.x . (latest)
When I packed my storage to the limit, at no point did I receive any form of notification. The only thing I noticed was that the quota bar in the web interface turned from blue to red. 

If this is not fixed, I would want to try to look for a fix. 

Thanks!

**Comment by camilasan** [comment]
This is done with https://github.com/nextcloud/desktop/pull/8462
